### PR TITLE
feat: add items page

### LINKED ISSUE #393: add items page
image

image

- similar to warehouse but with all items in aavegotchi game 
- card should include proposed effect , type, total qty , etc

### PR DESCRIPTION
**Pull request by dmutrokot**
Resolves #393

### CODE REVIEW

File: src/pages/Glossary/store/effects/glossary.effect.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    const wearablesCopy: Erc1155Item[] = _.cloneDeep(getState().glossary.wearables);
+
+    Promise.all([
+        TheGraphApi.getErc1155ListingsBatchQuery(wearablesIds, Erc1155Categories.Wearable, true, 'timeLastPurchased', 'desc'),
+        TheGraphApi.getErc1155ListingsBatchQuery(wearablesIds, Erc1155Categories.Wearable, false, 'priceInWei', 'asc')
+    ])
+    .then(([lastSoldListings, currentListings]: [Erc1155ListingsBatch, Erc1155ListingsBatch]) => {
+        const listingPrices: number[] = [];
+
+        Object.keys(lastSoldListings).forEach((key, index) => {
```
Comment (Vladislav94) [inline src/pages/Glossary/store/effects/glossary.effect.ts:21]: types

File: src/components/ItemCard/components/CardListing/CardListing.tsx [hunk 2]
```diff
@@ -50,14 +47,14 @@ export function CardListing({ className }: CardListingsProps) {
                 placement='top'
             >
                 {current.price === 0 ? (
-                    <div className={classNames(classes.listings, className)}>
+                    <div className={classNames(classes.listings)}>
```
[the remark this answers is not in the document]
Comment (dmutrokot) [reply src/components/ItemCard/components/CardListing/CardListing.tsx]: fixed, thanks

File: src/contexts/ClientContext.tsx [hunk 3]
```diff
@@ -166,19 +166,19 @@ export const ClientContextProvider = (props: any) => {
 
             // collect all equipped wearables
             allGotchis.forEach((item: any) => {
-                const equipped: any = item.equippedWearables.filter((item: any) => item > 0);
+                const equippedIds: any = item.equippedWearables.filter((item: any) => item > 0);
```
[the remark this answers is not in the document]
Comment (dmutrokot) [reply src/contexts/ClientContext.tsx]: fixed, thanks

File: src/pages/Glossary/components/GlossaryWearables/styles.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import { createStyles, makeStyles } from '@mui/styles';
+
+export const styles = makeStyles(() => createStyles({
+    glossaryWearablesContainer: {
+        padding: '56px 16px 12px'
+    },
+
```
[the remark this answers is not in the document]
Comment (dmutrokot) [reply src/pages/Glossary/components/GlossaryWearables/styles.ts]: fixed, thanks

File: src/contexts/ClientContext.tsx [hunk 5]
```diff
@@ -166,19 +166,19 @@ export const ClientContextProvider = (props: any) => {
...
 
-                for (const wearable of equipped) {
-                    const index: number = wearables.findIndex(item => item.id === wearable);
+                for (const wearableId of equippedIds) {
+                    const index: number = wearables.findIndex(item => item.id === wearableId);
```
[the remark this answers is not in the document]
Comment (dmutrokot) [reply src/contexts/ClientContext.tsx]: added `any`, can't set interface type for now

File: src/pages/Glossary/store/effects/glossary.effect.ts (hunk 1 above)
Comment (dmutrokot) [reply src/pages/Glossary/store/effects/glossary.effect.ts]: fixed, thanks

File: src/components/ItemCard/components/CardListing/CardListing.tsx [hunk 6]
```diff
@@ -50,14 +47,14 @@ export function CardListing({ className }: CardListingsProps) {
...
                         <Typography variant='subtitle2' className={classes.error}>No listings</Typography>
                     </div>
                 ) : (
                     <Link
                         href={`https://app.aavegotchi.com/baazaar/erc1155/${current.listing}`}
                         target='_blank'
-                        className={classNames(classes.listings, className)}
+                        className={classNames(classes.listings)}
```
[the remark this answers is not in the document]
Comment (dmutrokot) [reply src/components/ItemCard/components/CardListing/CardListing.tsx]: fixed, thanks

File: src/components/Content/ContentWrapper.tsx [hunk 7]
```diff
@@ -5,11 +5,11 @@ export function ContentWrapper({ children }: { children: JSX.Element[] }) {
...
-            <div className={classes.sidebar}>
+            <div className={classes.inner}>
                 {children[0]}
             </div>
 
-            <div className={classes.inner}>
+            <div className={classes.sidebar}>
                 {children[1]}
             </div>
         </div>
```
[the remark this answers is not in the document]
Comment (dmutrokot) [reply src/components/Content/ContentWrapper.tsx:15]: will be discussed later

**Review by Vladislav94 — APPROVED** [review]

**Pull request merged**